backend/Dockerfile.worker

```diff
@@ -41,7 +41,11 @@ RUN gem install bundler:2.3.21
 RUN export RUBY_VERSION=$(ruby -e "print RUBY_VERSION") && git clone https://github.com/intrigueio/intrigue-ident.git && cd intrigue-ident && git checkout ee119abeac20564e728a92ab786400126e7a97f0 && sed -i "s/2.7.2/$RUBY_VERSION/g" Gemfile && sed -i "s/2.7.2p114/$RUBY_VERSION/g" Gemfile.lock && bundle install --jobs=4
 RUN echo 'cd /app/intrigue-ident && bundle exec ruby ./util/ident.rb $@' > /usr/bin/intrigue-ident && chmod +x /usr/bin/intrigue-ident
 
+RUN export LD_LIBRARY_PATH=/opt/OpenBLAS/lib:$LD_LIBRARY_PATH
+
 # Install pe-source module
+# why we sync to a specific commit in cf-staging branch?
+# consider syncing to latest in a stable branch do avoid churn
 RUN git clone https://github.com/cisagov/pe-reports.git && cd pe-reports && git checkout c9cbbd73b22ef38cabe1da6ba50aeb2dc0be4f99 && pip install . 
```

Comment:
@aloftus23

Reply:
P&E set up a dedicated branch to be installed in Crossfeed's worker container because we only want to install necessary packages and it's setup to grab variables from the parameter store. 

The main pe-reports branch has a few functionalities including reporting, mailing, and data collection; we only want the data collection portion.  

I suggested a change that will make sure it's the latest.
```suggestion
RUN git clone -b cf-source-staging https://github.com/cisagov/pe-reports.git && cd pe-reports && pip install .
```

Reply:
Updated as per suggestion.